### PR TITLE
CLDR-13587 Update subdivisions (BRS A25.2)

### PR DESCRIPTION
**Pull request by macchiati**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13587
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: common/subdivisions/en.xml [hunk 1]
```diff
@@ -1650,6 +1650,7 @@ Generated by tool:	SubdivisionNode
 			<subdivision type="gbeln">East Lothian</subdivision> <!-- in gbsct : Scotland -->
 			<subdivision type="gbels">Outer Hebrides</subdivision> <!-- in gbsct : Scotland -->
 			<subdivision type="gbenf">Enfield</subdivision> <!-- in gbeng : England -->
+			<subdivision type="gbeng">England</subdivision> <!-- deprecated -->
```
Comment (pedberg-icu) [inline common/subdivisions/en.xml]: Hmm. Does the deprecation of gbeng, gbsct, gbwls have any impact on the use of those in emoji tag sequences? I thought those definitions depended on CLDR validity...

**Comment by macchiati** [comment]
It does, but they are still valid, just deprecated. However, we do need to
note this in the release notes.

From validity/subdivision.xml:

gbant gbard gbarm gbbla gbbly gbbnb gbcgv gbckf gbckt gbclr gbcsr gbdgn
gbdow gbdry gbeaw gbeng gbfer gbgbn gblmv gblrn gblsb gbmft gbmyl gbndn
gbnir gbnta gbnym gbomh gbsct gbstb gbukm gbwls ghba glqa gr01 gr03~7
gr11~7 gr21~4 gr31~4 gr41~4 gr51~9 gr61~4 gr71~3 gr81~5 gr91~4 gra1

From UTS #51

Let SD be the result of mapping each character in the tag_spec to a
character in [0-9a-z] by subtracting 0xE0000.

   1. SD must then be a specification as per [CLDR
   <http://unicode.org/reports/tr51/#CLDR>] of either a Unicode
   subdivision_id
   <http://unicode.org/reports/tr35/index.html#unicode_subdivision_id> or a
   3-digit unicode_region_subtag
   <http://unicode.org/reports/tr35/index.html#unicode_region_subtag>, and
   2. SD must have CLDR idStatus equal to "regular" or "deprecated".(

(I checked up on this when I saw the change, so we're thinking alike)

Mark

**Comment by pedberg-icu** [comment]
OK, thanks

**Comment by macchiati** [comment]
gbeng is deprecated? What replaces it?
markus

**Comment by macchiati** [comment]
Nothing; it just disappears. (It is an intermediate level in v36. )
Compare:
http://unicode.org/repos/cldr-aux/charts/36/supplemental/territory_subdivisions.html#gb
http://unicode.org/repos/cldr-aux/charts/37/supplemental/territory_subdivisions.html#gb

Mark

**Comment by macchiati** [comment]
But I'm looking into it now.

Mark

**Pull request merged**